src/registrar/management/commands/utility/terminal_helper.py

```diff
@@ -373,3 +373,22 @@ def print_to_file_conditional(print_condition: bool, filename: str, file_directo
             logger.info(f"{TerminalColors.MAGENTA}Writing to file " f" {filepath}..." f"{TerminalColors.ENDC}")
             with open(f"{filepath}", "w+") as f:
                 f.write(file_contents)
+
+    @staticmethod
+    def colorful_logger(log_level, color, message):
```

Comment:
CHANGE REQUEST:
I like it...let's do add parameter types and pass in enums instead of strings, though.  This has a high-reuse potential and if we upgrade it now we avoid mess later.